DigitalOffice/Tests/TimeManagementServiceTests/Repositories/WorkTimeRepositoryTests.cs

```diff
@@ -0,0 +1,123 @@
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using LT.DigitalOffice.TimeManagementService.Database;
+using LT.DigitalOffice.TimeManagementService.Database.Entities;
+using LT.DigitalOffice.TimeManagementService.Repositories;
+using LT.DigitalOffice.TimeManagementService.Repositories.Filters;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+
+namespace LT.DigitalOffice.TimeManagementServiceUnitTests.Repositories
+{
+    public class CreateWorkTimeTests
+    {
+        private TimeManagementDbContext dbContext;
+        private IWorkTimeRepository repository;
+
+        private static readonly Guid project1 = Guid.NewGuid();
+        private static readonly Guid project2 = Guid.NewGuid();
+
+        private static readonly Guid worker1 = Guid.NewGuid();
+        private static readonly Guid worker2 = Guid.NewGuid();
+
+        private readonly DbWorkTime testWorkTime1 = new DbWorkTime
+        {
+            Id = Guid.NewGuid(),
+            Title = $"WorkTime",
+            WorkerUserId = worker1,
+            ProjectId = project1,
+            Date = DateTime.Now.AddDays(-1), // 1 days ago
+            Time = new TimeSpan(6, 0, 0) // 0.75 days ago
+        };
+
+        private readonly DbWorkTime testWorkTime2 = new DbWorkTime
+        {
+            Id = Guid.NewGuid(),
+            Title = $"WorkTime",
+            WorkerUserId = worker2,
+            ProjectId = project1,
+            Date = DateTime.Now.AddDays(-0.9), // 0.9 days ago
+            Time = new TimeSpan(6, 0, 0) // 0.65 days ago
+        };
+
+        private readonly DbWorkTime testWorkTime3 = new DbWorkTime
+        {
+            Id = Guid.NewGuid(),
+            Title = $"WorkTime",
+            WorkerUserId = worker1,
+            ProjectId = project2,
+            Date = DateTime.Now.AddDays(-0.7), // 0.7 days ago
+            Time = new TimeSpan(6, 0, 0) // 0.45 days ago
+        };
+
+        [SetUp]
+        public void Setup()
+        {
+            var dbOptions = new DbContextOptionsBuilder<TimeManagementDbContext>()
+                                    .UseInMemoryDatabase(databaseName: "InMemoryDatabase")
+                                    .Options;
+            dbContext = new TimeManagementDbContext(dbOptions);
+            repository = new WorkTimeRepository(dbContext);
+        }
+
+        [TearDown]
+        public void Clear()
+        {
```

Comment:
What is cleared?

Reply:
DB. No?

Reply:
Agree on the naming. Should clarify what exactly do we clear (yeah I see that it uses DbContext so its a DB cleanup)
But naming conventions and overall method readability)

Reply:
Write CleanDb

Reply:
Renamed.